### PR TITLE
Feature/t21

### PR DESCRIPTION
**Pull request by linjoh92**
Add styling and set up some mockup structure with fake-data for the comemnt area.

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import './styles/App.css';
-import Header from './components/header'
+import Header from './components/headerr'
```
Comment (idamakela) [inline src/App.jsx]: A tiny typo, header is the correct spelling

**Review by idamakela — CHANGES_REQUESTED** [review]
tiny typo, see comment

File: src/App.jsx (hunk 1 above)
Comment (linjoh92) [reply src/App.jsx]: Fixed it!

**Review by idamakela — APPROVED** [review]
Perfect!!

**Pull request merged**